### DISCUSSION TITLE
regolith-gnome-flashback seems to fail on raspberry pi 5

### DISCUSSION THREAD

**Discussion by lektorrr** [regolith-linux/regolith-desktop#977] [Q&A]
Hello,

I don't know if this is the right place to bring this up, but:

I have installed regolith 3.1 on a raspberry pi 5 with Ubuntu 23.10, but when I try to log in, it just hangs for a brief moment, and then returns to the login screen.

How do I go about diagnosing this problem?  Are there any logs I can look through to find out where/what the failure is?  If it is a real problem, is there a place I can report it?

I have a raspberry pi 400 with regolith 3.1 on Ubuntu 23.10, which works fine, so it may be a quirk of the raspberry pi 5.

/Erasmus

**Comment by kgilmer** [comment]
Hi @lektorrr , hmm I have run Regolith on a variety of ARM hardware but not yet a RP5.  You could check `/var/log/syslog` for errors at the time the session init fails.

**Reply by kgilmer** [reply]
You might also try installing the prod version (3.0) to ensure you're not hitting some unknown bugs.

**Reply by lektorrr** [reply]
Hi kgilmer,

sadly, you can't install previous versions of Ubuntu on the pis, only 23.10 is available, and as I understand it, on 23.10 I have to go to regolith 3.1?  Hopefully, there will be an unofficial Debian 12 image at some point, and when/if that happens, I will try that out (I'd rather be running regolith 3.0 on debian 12, but that just doesn't seem possible right now).

I will check the syslog, as you suggest.

/Erasmus
